apps/aecontract/src/aect_call_tx.erl

```diff
@@ -37,15 +36,14 @@
          call_data/1]).
 
 
--define(CONTRACT_CALL_TX_TYPE, <<"contract_call">>).
 -define(CONTRACT_CALL_TX_VSN, 1).
 -define(CONTRACT_CALL_TX_FEE, 2).
 
--type call_tx() :: #contract_call_tx{}.
+-opaque tx() :: #contract_call_tx{}.
 
--export_type([call_tx/0]).
```

Comment:
This type is used somewhere though our Dialyzer config does not pick this. It looks like there are other cases like this.
I suggest merging this PR as is then type fixups (and Dialyzer improvement maybe) can find place in following PR.

----

Rough local testing result:
```
   0: Unknown type aect_call_tx:call_tx/0
   0: Unknown type aect_create_tx:create_tx/0
   0: Unknown type aens_claim_tx:claim_tx/0
   0: Unknown type aens_preclaim_tx:preclaim_tx/0
   0: Unknown type aens_transfer_tx:transfer_tx/0
   0: Unknown type aens_update_tx:update_tx/0
```

Reply:
I started it as https://www.pivotaltracker.com/story/show/155400295